fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedStoredFieldsFormatTest.java

```diff
@@ -52,18 +50,9 @@
 // sonarcloud doesn't seem to be able to detect the junit4 style of just having the method start with "test"
 @SuppressWarnings("java:S2187")
 public class LuceneOptimizedStoredFieldsFormatTest extends BaseStoredFieldsFormatTestCase {
+    private static final FDBDatabaseExtension dbExtension = new FDBDatabaseExtension();
```

Comment:
I don't see any reason why this isn't using `BaseIndexFileFormatTestCaseUtils` for setup/teardown

Reply:
Yeah, good point. I've consolidated that handling into `BaseIndexFileFormatTestCaseUtils`. There's something a little mismatched, in that the extension is static but is reset with every test. Theoretically, we'd want it to create a new one each time, but I'm not sure that is worth it given that I don't think we'll be parallelizing the JUnit4 tests.